ldap/spring-security-ldap.gradle

```diff
@@ -20,12 +20,15 @@ dependencies {
 		exclude(group: 'org.springframework.data', module: 'spring-data-commons')
 	}
 
+	testCompile project(':spring-security-test')
 	testCompile 'org.slf4j:jcl-over-slf4j'
 	testCompile 'org.slf4j:slf4j-api'
 }
 
 integrationTest {
-	include('**/ApacheDSServerIntegrationTests.class', '**/ApacheDSEmbeddedLdifTests.class')
+	include('**/ApacheDSServerIntegrationTests.class',
```

Comment:
Is this actually necessary since we have a separate source folder?

Reply:
The reason, as far as I can tell, is because `ApacheDSServerIntegrationTests` uses the `@Suite` annotation to invoke other dependent integration tests (in the same integration-test directory). If the tests are run independent from this orchestrating test, then they fail because they are dependent on the server that `ApacheDSServerIntegrationTests` starts up.

The password-modify tests are not written in this way, but need to be included in the list since we are already using the `include` directive.

Reply:
I'm not really a fan of it either. If you think it valuable, I'd recommend a ticket to rewrite the ldap tests to not depend on a parent integration test. Then we could remove this `include`.

Reply:
Ahh yes thanks for the reminder about that. Yes let's do that as a separate ticket.

Reply:
Done - https://github.com/spring-projects/spring-security/issues/5942